ArchiSteamFarm/Json/Steam.cs

```diff
@@ -391,7 +391,7 @@ internal class NumberResponse {
 		[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
 		internal sealed class RedeemWalletResponse : EResultResponse {
 			[JsonProperty(PropertyName = "detail", Required = Required.DisallowNull)]
-			internal readonly EPurchaseResultDetail? PurchaseResultDetail;
+			internal readonly EPurchaseResultDetail PurchaseResultDetail;
```

Comment:
As stated in the JSON itself, this property is not required, so it can be a `null` value. Removing nullable type from here is wrong.